genai-perf/genai_perf/parser.py

```diff
@@ -254,6 +254,8 @@ def _check_conditional_args(
                 f"The --generate-plots option is not currently supported with the {args.endpoint_type} endpoint type."
             )
 
+    if args.random_seed is None:
+        args.random_seed = random.randint(0, sys.maxsize)
```

Comment:
I think we can leave the seed as `None`. The python `random` will by default use system time for randomness: https://docs.python.org/3/library/random.html#random.seed